lib/exception_hunter/error_creator.rb

```diff
@@ -28,16 +35,26 @@ def call(tag: nil, **error_attrs)
           notify(error)
           error
         end
-      rescue ActiveRecord::RecordInvalid
-        false
       end
 
       private
 
+      def log_error(tag, error_attrs)
+        if async_logging?
+          ExceptionHunter::AsyncLoggingJob.perform_later(tag, error_attrs)
```

Comment:
Are we sending the `occurend_at` here? If not that time would be offset when the job is called